### ISSUE TITLE
Apply locale to computed text

### ISSUE THREAD

**Issue by amyjko** [wordplaydev/wordplay#487]
## What's the problem?

There are many ways to generate text, but no way to apply a locale to computed text once computed. This means that locales can only be applied to static and templated text, but not text generate through more sophisticated computation.

## What's the design idea?

Add a feature to the language, or perhaps an API on `text` values, to apply a locale to their value.

## Design specification

_Reporters can leave this blank. Designers should write a detailed, precise description of what is to be built. If images are necessary to convey this precisely, make sure to include image descriptions, so everyone can see what is in the image. Do not link to external documents; everything should be embedded here. Once the design is approved, we will remove the `needs design` tag and replace it with the `buildable` tag, signalling that it can be built._

**Comment by amyjko** [comment]
Duplicate of #526.
